toolchain/lower/function_context.h

```diff
@@ -126,6 +139,47 @@ class FunctionContext {
   auto FinishInit(SemIR::TypeId type_id, SemIR::InstId dest_id,
                   SemIR::InstId source_id) -> void;
 
+  auto AddCallToCurrentFingerprint(SemIR::FunctionId function_id,
+                                   SemIR::SpecificId specific_id) {
+    if (!function_fingerprint_) {
+      return;
+    }
+    current_fingerprint_.relevant = true;
+
+    RawStringOstream os;
+    // TODO: Replace with id that is translation unit independent.
+    function_id.Print(os);
```

Comment:
Maybe replace with directly dropping in the index, rather than using `Print`? I think `Id::Print` should be regarded as a debug-only feature.